public/controlpanel.php

```diff
@@ -225,7 +231,24 @@ function confirmEmailChange(event) {
                     </tr>
                     </tbody>
                 </table>
-                <img id='loadingicon' style='opacity: 0; float: right;' src='<?= asset('assets/images/icon/loading.gif') ?>' width='16' height='16' alt='loading icon'/>
+                <img id='loadingicon-1' style='opacity: 0; float: right;' src='<?= asset('assets/images/icon/loading.gif') ?>' width='16' height='16' alt='loading icon'/>
+            </div>
+
+            <div class='component'>
+                <h3>Settings</h3>
```

Comment:
Since you're adding a new section here, you should probably move the Mature Content warning into it.

Reply:
Agreed, should be addressed in the latest change.

Reply:
Sounds reasonable. Maybe `Site_SuppressMatureContentWarning` to match the text you used on the settings page?

Reply:
Handled in https://github.com/RetroAchievements/RAWeb/pull/1656/commits/dfebc91447b224d96acc164d736ea0df5fec0ea3. 

Note that in this commit I removed the value from _NotificationPreferences.php_ altogether [here](https://github.com/RetroAchievements/RAWeb/pull/1656/files#diff-90a5d2802a1107b7dd3334575f3a229d0347615bcc109d483c6aef5bb748154f). I don't believe this is something we'd want to send out user notifications for, though I am happy to back out that change.